zzzz_modular_occulus/code/game/jobs/job/guild.dm

```diff
@@ -15,3 +15,7 @@
 		STAT_TGH = 10,
 		STAT_VIG = 10
 	)
+	perks = list(/datum/perk/deep_connection, /datum/perk/oddity/market_prof)
+
+/datum/job/mining
```

Comment:
Union Techs and Miners shouldn't start with this perk.
Miners have even less reason to have this. Technically they shouldn't be selling or buying much at all.